dm/config/task.go

```diff
@@ -350,7 +352,7 @@ func (c *TaskConfig) adjust() error {
 		switch c.TaskMode {
 		case ModeFull, ModeAll:
 			if inst.Meta != nil {
-				log.Warnf("[config] mysql-instance(%d) set meta, but it will not be used for task-mode %s.\n for Full mode, incremental sync will never occur; for All mode, the meta dumped by MyDumper will be used", i, c.TaskMode)
+				log.L().Warn("metadata setted will not be used. for Full mode, incremental sync will never occur; for All mode, the meta dumped by MyDumper will be used", zap.Int("mysql instance", i), zap.String("task mode", c.TaskMode))
```

Comment:
```suggestion
				log.L().Warn("metadata will not be used. for Full mode, incremental sync will never occur; for All mode, the meta dumped by MyDumper will be used", zap.Int("mysql instance", i), zap.String("task mode", c.TaskMode))
```
The past participle of the word `set` is still `set`